repository/src/main/java/org/apache/atlas/repository/store/graph/v2/bulkimport/MigrationImport.java

```diff
@@ -92,6 +97,13 @@ public EntityMutationResponse run(EntityImportStream entityStream, AtlasImportRe
         return ret;
     }
 
+    @Override
+    public TypesUtil.Pair<EntityMutationResponse, Float> run(AtlasEntity.AtlasEntityWithExtInfo entityWithExtInfo, EntityMutationResponse ret,
+                                                             AtlasImportResult importResult, Set<String> processedGuids, int entityStreamPosition,
+                                                             int streamSize, float currentPercent, List<String> residualList) throws AtlasBaseException {
+        return null;
```

Comment:
`throw new NotImplementedException("MigrationImport: entity import not implemented");`